tests/countries/test_netherlands.py

```diff
@@ -145,25 +143,31 @@ def test_2017(self):
     def test_2020(self):
         self.assertHolidays(
             ("2020-01-01", "Nieuwjaarsdag"),
-            ("2020-04-10", "Goede Vrijdag"),
             ("2020-04-12", "Eerste paasdag"),
             ("2020-04-13", "Tweede paasdag"),
             ("2020-04-27", "Koningsdag"),
-            ("2020-05-05", "Bevrijdingsdag"),
             ("2020-05-21", "Hemelvaartsdag"),
             ("2020-05-31", "Eerste Pinksterdag"),
             ("2020-06-01", "Tweede Pinksterdag"),
             ("2020-12-25", "Eerste Kerstdag"),
             ("2020-12-26", "Tweede Kerstdag"),
         )
 
+    def test_optional_2023(self):
```

Comment:
Let's stick with this naming format:
```suggestion
    def test_2023_optional(self):
```